.github/dependabot.yml

```diff
@@ -0,0 +1,22 @@
+version: 2
+updates:
+  - package-ecosystem: "pip"
+    directory: "/"
+    schedule:
+      interval: "daily"
+      time: "07:00"
+    versioning-strategy: auto
```

Comment:
Maybe add a comment about this? Is this for how to interpret dependencies versions or if to update to minor or major versions? Or for how the project version itself? Maybe add a comment (with a link to the docs)?

Reply:
This is the default versioning-strategy. For apps, the version requirements are increased. For libraries, the range of versions is widened. It isn't required but I added explicitly in case we need to change it. I'll add a reference to the docs

Reply:
Maybe add a comment saying just that? (and add the link to the docs?)

Reply:
Also, how does it know that a repo is an app or a lib? No dependencies lock file means is a lib?

Reply:
It isn't precisely specified in the documentation as it depends on the ecosystem. My understanding is it depends on an entry point in the manifest file(s) to consider the repo an app